src/sysrepo.h

```diff
@@ -219,12 +219,17 @@ typedef struct sr_error_info_s {
     size_t err_count;    /**< Error message count. */
 } sr_error_info_t;
 
+/**
+ * @brief Connection ID.
+ */
+typedef uint32_t sr_cid_t;
+
```

Comment:
The type is no longer needed publicly, right?

Reply:
Correct.  Moved to common.h